libtenzir/include/tenzir/data_builder.hpp

```diff
@@ -441,8 +441,8 @@ class node_object : public node_base {
   auto record() -> node_record*;
   auto list() -> node_list*;
 
-  node_object() : data_{std::in_place_type<caf::none_t>} {
```

Comment:
Does this need a changelog entry?

Reply:
Why do you think it should get one?

Reply:
Oh, it didn't compile in the first place. I read this as if it was adding data of a wrong type or sth like that. Ignore the comment then.